MF-1580 - Influxdb Writer changes format of update-time to string

Hasan98-git opened the pull request:
### What does this do?
This will stop influxdb-writer service from changing the datatype of update-time variable to string

### Which issue(s) does this PR fix/relate to?
Resolves #1580

### List any changes that modify/break current functionality
It won't break current functionality

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

Commit:
- MF-1580
- Modified consumers/writers/influxdb/fields.go
- influxdb-writer used to update data type of update-time to string
- Commented line 12 of consumers/writers/influxdb/fields.go to resolve
  uneccessary data type conversion issue

Signed-off-by: Hasan Tariq <hasantariqashraf@gmail.com>

manuio commented on consumers/writers/influxdb/fields.go:
```diff
@@ -9,7 +9,7 @@ import (
 
 func senmlFields(msg senml.Message) fields {
-	updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
+	// updateTime := strconv.FormatFloat(msg.UpdateTime, 'f', -1, 64)
```
You can simply remove it

Hasan98-git replied:
Okay, but I have checked that the build fails, I thing i will need to remove the strconv package also and replace retur
n value of updateTime with msg.UpdateTime

Hasan98-git committed:
- MF-1580
- Removed strconv package from consumers/writers/influxdb/fields.go since it is no longer needed
- Removed line 12 from consumers/writers/influxdb/fields.go
- Replaced retrun value of updateTime with msg.UpdateTime (line 16 in
  fields.go)

Signed-off-by: Hasan Tariq <hasantariqashraf@gmail.com>

dborovcanin committed:
Fix InflxuDB readers

Signed-off-by: dusanb94 <dusan.borovcanin@mainflux.com>

manuio reviewed (approved):
LGTM
